README.md

```diff
@@ -7,22 +7,38 @@ Capistrano ForkCMS - Easy deployment of ForkCMS 5+ apps with Ruby over SSH.
 
 ## Installation
 
-Add this line to your application's Gemfile:
+First, we create a Gemfile by executing:
```

Comment:
Not relevant, If you don't know how to use Gemfile's you probably won't have bundler installed.